Document setting bond options (#221)
linux-system-roles/network: README.md

    If set, NetworkManager will ensure the domain name of the EAP server certificate matches this string.

### `bond`

> The `bond` option sets the bond.options field for a bonded interface.
> 
> The `bond` setting supports the following options:

Reviewer: ```suggestion
The `bond` setting configures the  options of bonded interfaces (type `bond`). It supports the following options:
```

  * `mode`

>     Bonding mode.  See the [kernel documentation](https://www.kernel.org/doc/Documentation/networking/bonding.txt) or your vendor documentation for valid values.  NetworkManager defaults to `balance-rr`.

Reviewer: Please wrap this line at about 88 characters.
